gradle/update-copyrights.gradle

```diff
@@ -6,7 +6,7 @@ if (gitPresent) {
 
 ext {
 	if (gitPresent) {
-		modifiedFiles = files(grgit.status().unstaged.modified).filter{ f -> f.name.endsWith('.java') }
+		modifiedFiles = providers.provider { files(grgit.status().unstaged.modified).filter{ f -> f.name.endsWith('.java') } }
```

Comment:
This was in there previously but I removed as `gradle build` was failing on it. I thought maybe it was from Gradle 8 but did not look too much into it.